### DISCUSSION TITLE
Assign option.none to a variable

### DISCUSSION THREAD

**Discussion by LCRERGO** [vlang/v#20205] [Questions and Answers] [answered]
Hi, I have been trying to define a variable as none where its value is of type Option<Terrain> where Terrain is a named struct. But when I try to assign the variable with ```current_terrain = option.none``` I get ```unexpected keyword `none`, expecting name```. Is it possible to do something like that? I couldn't find similar examples in the documentation

**Comment by JalonSolov** [comment, marked as answer]
Just `none`, not `option.none`.

**Reply by LCRERGO** [reply]
Thanks
